packages/govtnz-ds-website/src/commons/styles/elements-global.scss

```diff
@@ -24,6 +24,10 @@ html.theme-default {
   --g-theme-font-family: National2, Arial, sans-serif;
 }
 
+.theme-font-arial {
+  --g-theme-font-family: Arial, sans-serif;
+}
```

Comment:
@holloway  this is what I did with overriding the other font. Not sure if this is where you would like the class, seemed most relevant place to me .

Reply:
Should we apply this class on the `<Example>` or `<ExampleContainer>` component? It seems we'd always want to reset it there

Reply:
@holloway  do you mean add it here ?

```
  <ExampleContainer className="theme-font-arial">
      <Row>
        <Column className="" xs="12" sm="12" md="9" lg="12">
          <ContactusForm />
        </Column>
      </Row>
    </ExampleContainer>
```

Reply:
@Juliet-Brown  I mean that the CSS should just be added here because every example should use Arial and should not inherit the National2 font.

https://github.com/GOVTNZ/govtnz-design-system/blob/master/packages/govtnz-ds-website/src/commons/styles/components-example-container.scss

```css
.example__container {
  /* ...other stuff... */
  --g-theme-font-family: Arial, sans-serif;
}
```

Reply:
@holloway  Have updated.